### ISSUE TITLE
Tag a new version ⚠️

### ISSUE THREAD

**Issue by tristanbes** [humanmade/S3-Uploads#283]
Hello,

Last tagged version is v2 in **march**, there is critical FIXES which has been made and not available in v2 (for example https://github.com/humanmade/S3-Uploads/pull/255)

Can you tag a new version so I don't have to require `dev-master` branch using composer ? Thanks.

**Comment by tristanbes** [comment]
Please, can we get the tag already ? (and not a RC) :)

**Comment by Kocal** [comment]
It would be so nice to have a new tagged release, at least for #255 that is really cumbersome... :confused:

Thanks!

**Comment by tristanbes** [comment]
ping @joehoyle // @rmccue

**Comment by joehoyle** [comment]
Waiting on https://github.com/humanmade/S3-Uploads/pull/293 to go with version 2.

**Comment by joehoyle** [comment]
This is now done.

**Comment by Kocal** [comment]
Thanks!
